agenta-cli/agenta/cli/helper.py

```diff
@@ -1,11 +1,35 @@
-from pathlib import Path
-from typing import Any, List, MutableMapping
-import click
+import sys
 import toml
+import click
+import questionary
+from pathlib import Path
 from agenta.client import client
+from typing import Any, List, MutableMapping
 from agenta.client.api_models import AppVariant
 
 
+def get_api_key():
+    agenta_dir = Path.home() / ".agenta"
+    credentials_file = agenta_dir / "credentials"
+
+    if credentials_file.is_file():
+        with credentials_file.open("r") as f:
```

Comment:
please use toml instead as done in the rest of the cli instead of using our own format

Reply:
@mmabrouk 
I considered that we were persisting this to the local storage of the users computer, and that we may reuse this file to store other credentials in the future and decided to go with this approach. What do you think?

Reply:
yes, totally agree. I just mean you should use a toml file when saving the credentials and not create your own format.

Reply:
basically use `toml.load` and `toml.dump` instead of this

Reply:
@mmabrouk 
toml only works with .toml files, the file type of credentials is not a toml file as as such `toml.load` and `toml.dump` won't work with it. 

Would you rather I change it to `credentials.toml`?

Reply:
or simply config.toml so that we can use it for other things later